src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -139,9 +140,7 @@ private HostAndPort initSentinels(Set<String> sentinels, final String masterName
 
       log.fine("Connecting to Sentinel " + hap);
 
-      Jedis jedis = null;
-      try {
-        jedis = new Jedis(hap.getHost(), hap.getPort());
+      try (Jedis jedis = new Jedis(hap.getHost(), hap.getPort())) {
```

Comment:
@marcosnils I have to disagree here . As you already upgraded the JDK to 1.7 in the master branch, I believe that as a principal all new pieces of code written should be employ latest writing techniques , such  as try-with-resources. People looking to resolve a rather minor issue such as the above  should upgrade to newer versions , which is also good practice.